plutus-core/plutus-ir/test/prettyprintingReadable/basic.golden

```diff
@@ -1 +1 @@
-/\a -> \(x : a) -> x
\ No newline at end of file
+/\a_0 -> \(x_1 : a_0) -> x_1
```

Comment:
hmmm, should this change go in separately? not a big deal

Reply:
Counterexamples were completely useless without it.

Reply:
I just mean you could PR it first to make this smaller, but nbd

Reply:
It's a pretty tiny change in an otherwise 2k lines PR, so I'm not gonna bother.